resources/views/product.blade.php

```diff
@@ -0,0 +1,35 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Routing</title>
+</head>
+<body>
+    @isset($products)
```

Comment:
This is not required to check the value. Alternatively, if you want to check whether $products is an object or an array, you can do that. If it is, then proceed to loop; otherwise, exit the code.